mkosi/manifest.py

```diff
@@ -64,6 +64,20 @@ def report(self) -> str:
             t += f"""\nChangelog:\n{self.changelog}\n"""
         return t
 
+def parse_pkg_desc(f: Path) -> Tuple[str, str, str, str]:
+    with f.open() as desc:
```

Comment:
```suggestion

def parse_pkg_desc(f: Path) -> Tuple[str, str, str, str]:
    name = version = base = arch = ""
    with f.open() as desc:
```
Fixes the error pyright is complaining about and makes the code not break in case of a broken `desc` file.